src/components/FormToggle.vue

```diff
@@ -118,6 +135,9 @@ export default {
                 lineHeight: px(this.height),
             };
         },
+        showFalseInput() {
+            return this.falseValue && !this.value;
```

Comment:
this won't work

have you even test it ? :) 

since falseValue is `0` (number) it will never showFalseInput, test for null `this.falseValue == null` so default it will send `'0'` (string) and if we don't want to send anything we will pass `:false-value="null"` but I think that won't be used often

add <form> with <form-toggle> and try submit true/false if you see in formData `1`/`0`